loader/settings.c

```diff
@@ -302,6 +392,17 @@ void log_settings(const struct loader_instance* inst, loader_settings* settings)
         loader_log(inst, VULKAN_LOADER_DEBUG_BIT, 0, "Control: %s",
                    loader_settings_layer_control_to_string(settings->layer_configurations[i].control));
     }
+    if (settings->additional_driver_count > 0) {
+        loader_log(inst, VULKAN_LOADER_DEBUG_BIT, 0, "----");
+        loader_log(inst, VULKAN_LOADER_DEBUG_BIT, 0, "Use Additional Drivers Exclusively = %s",
```

Comment:
@charles-lunarg I am not convinced it's a debug level message. For me "debug level" is for the Loader developer but this is a relevant information for application developer. So I think it's a VULKAN_LOADER_NOFICATION_BIT level message.